rootfs/etc/haproxy/template/haproxy.tmpl

```diff
@@ -127,7 +127,7 @@ frontend httpfront
 ###### HTTPS frontend (tcp mode)
 ######
 frontend httpsfront
-    bind *:443
+    bind *:443{{ if $cfg.UseProxyProtocol }} accept-proxy{{ end }}
```

Comment:
No there should be no problem with adding `accept-proxy` to the stats binding as long as it is in addition to the proposed monitor-uri binding from #21 and the latter does not have `accept-proxy` (so Kubernetes can call it without the proxy protocol).

Thanks for the merge!